components/FormikFields/PaperInputPicker/index.js

```diff
@@ -62,6 +64,10 @@ const PaperInputPicker = ({
   const [pictureUris, setPictureUris] = React.useState({});
   const [image, setImage] = React.useState(null);
 
+  const [numberedQestions, setNumberedQuestions] = React.useState({});
+  const [questionsToRepeat, setQuestionsToRepeat] = React.useState([]);
```

Comment:
@jamccomb92 looking here a bit more closely, these aren't used in the `PaperInputPicker` correct? Can these be moved to the Looper component as well?

I see only the `additionalQuestions` state needing to stay here correct?